### PR TITLE
[sdk] update to Gecko SDK 3.2.1

### PR DESCRIPTION
**Pull request by lmnotran**
This PR updates the `sdk_support` submodule to the Gecko SDK 3.2.1 release. It also updates the `openthread` submodule

## Testing
I was able to form a network using `ot-cli-ftd` on two `brd4161a` boards and pinged both directions successfully.

The sleepy-demo also works as LEDs work as expected and the MTD is sleeping properly 
Screen Shot 2021-07-09 at 4 25 19 PM

### CODE REVIEW

**Comment by lmnotran** [comment]
@googlebot I consent.

**Comment by schinchani** [comment]
@googlebot I consent.

**Comment by Tinguely** [comment]
LGTM

File: src/src/openthread-core-efr32-config.h
Comment (schinchani) [inline src/src/openthread-core-efr32-config.h:68]: Hi @lmnotran  - Can we delete this change which says "currently not supported" ? This option is supported in GitHub.

**Comment by hemanth-silabs** [comment]
LGTM

**Comment by hemanth-silabs** [comment]
@googlebot I consent.

**Review by hemanth-silabs — APPROVED** [review]

**Review by schinchani — APPROVED** [review]
Besides the small comment clean-up, I suggested earlier, the other changes LGTM

**Comment by lmnotran** [comment]
@jwhui This should be ready to merge after CI passes

**Comment by jwhui** [comment]
@romacdon , would you like to review this PR? You are tagged as a reviewer :)

**Comment by schinchani** [comment]
> @romacdon , would you like to review this PR? You are tagged as a reviewer :)

Hi @jwhui , @romacdon may be tied up with other tasks. He had asked us to proceed with the merge if things look okay. He will review as soon as he has bandwidth (even if it is post-merge).

**Review by jwhui — APPROVED** [review]

**Comment by gabekassel** [comment]
@lmnotran would it be possible to include a link to the release notes for major updates like this in the future? hugely helpful to understand what's changing for those of us that don't love Simplicty Studio. Or is this somewhere in the contents?

https://www.silabs.com/documents/public/release-notes/open-thread-release-notes-1.2.1.0.pdf

**Pull request merged**
